src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -92,9 +92,9 @@ void GcsServer::Start() {
       new rpc::ErrorInfoGrpcService(main_service_, *error_info_handler_));
   rpc_server_.RegisterService(*error_info_service_);
 
-  worker_info_handler_ = InitWorkerInfoHandler();
+  gcs_worker_manager_ = InitGcsWorkerManager();
```

Comment:
now we call a class `handler` if it is used for info getting and setting, while `manager` doing something more.

Reply:
it also hadles `ReportWokerFailure` request and `RegisterWorker` request.